pages/k8s/validation.md

```diff
@@ -31,7 +31,7 @@ through the charm's actions.
 Add the charm to your cluster:
 
 ```bash
-juju deploy cs:~containers/kubernetes-e2e --constraints mem=4G --channel edge
+juju deploy kubernetes-e2e --constraints mem=4G --channel edge
```

Comment:
why are we using --edge? i mean, I know that's what was there, but why?

Reply:
no reason.. good catch.  updated to drop that `--channel` arg.